### PR TITLE
InspectOptions.lockedVersions

### PR DESCRIPTION
**Pull request by isoos**

### CODE REVIEW

File: lib/src/package_analyzer.dart [hunk 1]
```diff
@@ -54,6 +59,7 @@ class InspectOptions {
     this.isInternal = false,
     this.lineLength,
     this.analysisOptionsUri,
+    this.lockedVersions,
```
Comment (sigurdm) [inline lib/src/package_analyzer.dart:62]: Could we give this an empty map as a default, that would save a null check later.

**Comment by sigurdm** [comment]
Sorry for only seeing this now.

**Comment by isoos** [comment]
> would be awesome with a test.

maybe as part of end2end tests?

**Comment by isoos** [comment]
This is now obsolete I think.

**Pull request closed without merging**